sapl/relatorios/views.py

```diff
@@ -782,63 +733,41 @@ def get_sessao_plenaria(sessao, casa):
                 votos_materia_od.append(vp_od)
 
-        dic_votacao_vot_nom = {
-            'titulo': t_materia,
-            'votos': votos_materia_od
-        }
-        lst_votacao_vot_nom.append(dic_votacao_vot_nom)
+        lst_votacao_vot_nom.append({
+            "titulo": t_materia,
+            "votos": votos_materia_od
+        })
 
     # Lista dos oradores da Ordem do Dia
     lst_oradores_ordemdia = []
 
-    oradores_ordem_dia = OradorOrdemDia.objects.filter(
-        sessao_plenaria=sessao
-    ).order_by('numero_ordem')
+    oradores_ordem_dia = OradorOrdemDia.objects.filter(sessao_plenaria=sessao).order_by('numero_ordem')
 
     for orador_ordemdia in oradores_ordem_dia:
-        parlamentar_orador = Parlamentar.objects.get(
-            id=orador_ordemdia.parlamentar.id
-        )
-
-        sigla_partido = Filiacao.objects.filter(
-            parlamentar=parlamentar_orador
-        ).first()
-
-        if not sigla_partido:
-            sigla_p = ""
-        else:
-            sigla_p = sigla_partido.partido.sigla
-
-        dic_oradores_ordemdia = {
-            'num_ordem': orador_ordemdia.numero_ordem,
-            'nome_parlamentar': parlamentar_orador.nome_parlamentar,
-            'observacao': orador_ordemdia.observacao,
-            'sigla': sigla_p
-        }
-        lst_oradores_ordemdia.append(dic_oradores_ordemdia)
+        parlamentar_orador = Parlamentar.objects.get(id=orador_ordemdia.parlamentar.id)
+        sigla_partido = Filiacao.objects.filter(parlamentar=parlamentar_orador).first()
+
+        lst_oradores_ordemdia.append({
+            "num_ordem": orador_ordemdia.numero_ordem,
+            "nome_parlamentar": parlamentar_orador.nome_parlamentar,
+            "observacao": orador_ordemdia.observacao,
+            "sigla": "" if not sigla_partido else sigla_partido.partido.sigla
+        })
 
     # Lista dos oradores nas Explicações Pessoais
     lst_oradores = []
-    for orador in Orador.objects.filter(
-            sessao_plenaria=sessao).order_by('numero_ordem'):
-        for parlamentar in Parlamentar.objects.filter(
-                id=orador.parlamentar.id):
-            dic_oradores = {}
-            dic_oradores["num_ordem"] = orador.numero_ordem
-            dic_oradores["nom_parlamentar"] = parlamentar.nome_parlamentar
-            partido_sigla = Filiacao.objects.filter(
-                parlamentar=parlamentar).first()
-            if not partido_sigla:
-                sigla = ''
-            else:
-                sigla = partido_sigla.partido.sigla
-            dic_oradores['sgl_partido'] = sigla
-            lst_oradores.append(dic_oradores)
+    for orador in Orador.objects.filter(sessao_plenaria=sessao).order_by('numero_ordem'):
+        for parlamentar in Parlamentar.objects.filter(id=orador.parlamentar.id):
+            partido_sigla = Filiacao.objects.filter(parlamentar=parlamentar).first()
```

Comment:
Já que a ideia é otimizar esse código então, não precisa fazer tanta consulta direta ao banco (e se a relação entre orador e parlamentar é de um pra um não precisa de um for loop). Desta forma, este código, poderia ser assim, não?
```suggestion
    for orador in Orador.objects.select_related('parlamentar').filter(sessao_plenaria=sessao).order_by('numero_ordem'):
        parlamentar = orador.parlamentar
        partido_sigla = orador.parlamentar.filiacao_set.first()
```